docs/rest-api/source/includes/wp-api-v3/payouts.md

```diff
@@ -1,8 +1,8 @@
-# Deposits
+# Payouts
 
-The Deposits API endpoints provide access to an account's deposits data, including an overview of account balances, deposit schedule and deposit history.
+The Payouts API endpoints provide access to an account's payouts data, including an overview of account balances, payout schedule and payout history.
```

Comment:
Since we still have references to `deposit` in the API responses, it may be valuable to add a note here like "Note: payouts have historically been referred to as deposits, therefore these terms may be used interchangeably in response objects."

<img width="483" alt="image" src="https://github.com/user-attachments/assets/422f5ae4-74cc-4fc5-b848-e38e17fda997">

Reply:
Thanks for the review and the additional tip. I have [58b1fc1](https://github.com/Automattic/woocommerce-payments/pull/9669/commits/58b1fc11092446e0e8dce7fb58b47f5f46840ba1) as per your valid suggestion! Thanks.

![image](https://github.com/user-attachments/assets/e16b72a2-2eec-4727-ab2f-33694539b7b3)